Answers.md

```diff
@@ -0,0 +1,15 @@
+1.  Describe the following: `DataBase`, `Collection` , `Document`
+
+`DataBase` is either hardware or software served for the purposes of storing data. 
+
+`Collection` is a table of data organized by some criteria. 
```

Comment:
Be careful using the word table.  Tables are generally associated with SQL databases.